src/components/FooterWrapper.tsx

```diff
@@ -0,0 +1,11 @@
+import { FooterCfGov } from 'design-system-react';
+
+function FooterWrapper(): JSX.Element {
+  return (
+    <div className='mt-[3.75rem]'>
+      <FooterCfGov />
```

Comment:
Heyo, this causes some weirdness @shindigira by pushing up all the content, like here on the unauthenticated landing page:

_Before_
![Screenshot 2024-03-28 at 5 14 04 PM](https://github.com/cfpb/sbl-frontend/assets/19983248/e4039b89-9119-498e-b923-2be21c49ba72)


_After_
![Screenshot 2024-03-28 at 5 14 15 PM](https://github.com/cfpb/sbl-frontend/assets/19983248/c9633089-980c-40d5-b1ff-2ca879ab41c2)

It also increases the spacing at the bottom too much for other pages:
![Screenshot 2024-03-28 at 5 19 29 PM](https://github.com/cfpb/sbl-frontend/assets/19983248/a3cf23db-d825-4fd6-80de-f5b0177e8b0e)